sql/modules/Reconciliation.sql

```diff
@@ -93,6 +115,30 @@ $$Sets which lines of the report are cleared.$$;
 CREATE OR REPLACE FUNCTION reconciliation__delete_my_report(in_report_id int)
 RETURNS BOOL AS
 $$
+DECLARE
+    failed BOOL;
+BEGIN
+    SELECT approved INTO failed FROM cr_report
```

Comment:
No need for the `failed` variable here (which is unused in the result):

`PERFORM * FROM cr_report ...`

Reply:
Fixed